src/mayonnaise-or-aioli.md

```diff
@@ -0,0 +1,30 @@
+# Mayonnaise or aioli
+
+![Aioli](pix/mayonnaise-or-aioli.webp)
+
+Aioli is just mayonnaise with garlic. This recipe uses raw eggs, so if that is sketchy in your region use pasteurized eggs 🥚.
+
+- ⏲️ Prep time: 2 min
+- 🍳 Cook time: 1 min
+
+## Ingredients
+
+- 1 egg, preferably at room temperature
+- 2.5 dl/ 1 cup neutral oil
+- 1-2 tbsp vinegar or lemon juice.
+- 1 tbsp pickle juice (optional)
+- 2-4 cloves of garlic for aioli, don't include for mayonnaise
+- 1 tsp dijon mustard (optional)
+
+## Directions
+
+1. Place egg, vinegar, mustard a pinch of salt and pepper in a beaker or narrow and tall container. Add diced garlic for aioli.
+2. Add oil as the very last ingredient. It should be on the top of the other ingredients, or the mayonnaise/aioli could seperate.
+3. Place hand blender at the very bottom. Dont start it yet.
+4. Use full power, and work your way upwards slowly until all the oil is mixed in
+
+## Contribution
+
+- brox - xmr: `8AaUvC4JPfTVgy66FLd8mVfsPwT7RXJuPK2xySoKnFEhDZPNafpgn6rh8VPBVauyNP17EQQ1HGhMCZdtUJNoD7xnVt1cSLo`
+
+;tags: mayonnaise aioli
```

Comment:
```suggestion
;tags: sauce basic aioli
```
Again, sauce and basic would probably be good here. Mayo is a bit generic as a tag I think.